apps.json

```diff
@@ -1789,6 +1789,13 @@
         ],
         "url": "https://github.com/YunoHost-Apps/lemmy_ynh"
     },
+    "lidarr": {
```

Comment:
🤔 Lidarr should come after Libreto... right?

Reply:
Shame !🔔